README.md

```diff
@@ -23,3 +23,8 @@ Look for an interesting feature or defect [under issues](https://github.com/mbed
 
 ## Develop
 Information for setting up a development environment, running the tests or creating a release build [can be found in the developers guide.](docs/DEVELOPERS-GUIDE.md)
+
+## Known Issues
+Drag-n-drop program or application flashing fails for the first time on Windows machine with failure message "File sent out of order by PC. Target might not be programmed correctly". Windows tries to create "System Volume Information" folder while sending program data, which results in out of order error. Solution to this issue is:
```

Comment:
Does this reliably happen the first time for you? For me, it seems to happen randomly. If you plug in your board and wait does it make this less likely to happen? Also, is this only for binary files, or does this happen to hex files as well?

Reply:
This should happen just as often with hex files if the file is sent out of order. @deepikabhavnani do you still have the logs of this occurring? I would like to investigate this further. It may be something that can be improved in the DAPLink firmware, rather than requiring a user to change settings on their machine.